src/NitroSharp/Graphics/Shaders/transition.vert

```diff
@@ -11,10 +11,12 @@ layout(location = 2) in vec4 vs_Color;
 
 layout(location = 0) out vec4 fs_Color;
 layout(location = 1) out vec2 fs_TexCoord;
+layout(location = 2) out vec2 fs_MaskCoord;
 
 void main()
 {
     gl_Position = _ViewProjection * vec4(vs_Position, 0, 1);
     fs_Color = vs_Color;
     fs_TexCoord = vs_TexCoord;
+    fs_MaskCoord = (gl_Position.xy / gl_Position.w + 1.0) / vec2(2.0, -2.0);
```

Comment:
Why was this change necessary? Can you provide an example of what's wrong with the current implementation?